### PR TITLE
[ticket/16724] Add some template events

### PR DESCRIPTION
**Pull request by Dark1z**
Checklist:

- [x] Correct branch: master for new features; 3.3.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-16724

### CODE REVIEW

**Comment by rxu** [comment]
Twig syntax, please 😃

File: phpBB/docs/events.md [hunk 1]
```diff
@@ -986,6 +986,34 @@ mcp_forum_actions_before
 * Since: 3.1.11-RC1
 * Purpose: Add some information before actions fieldset
 
+mcp_forum_last_post_author_username_append
+===
+* Locations:
+    + styles/prosilver/template/mcp_forum.html (2)
```
Comment (rxu) [inline phpBB/docs/events.md:992]: What does `(2)` mean here and below?

Comment (Dark1z) [reply phpBB/docs/events.md:992]: > What does `(2)` mean here and below?

Occurrence of the event in that file.

**Comment by Dark1z** [comment]
> Twig syntax, please :)

Ahh!!! Ok...

File: phpBB/docs/events.md (hunk 1 above)
Comment (rxu) [reply phpBB/docs/events.md:992]: Doubt it's allowed, but it's up to devs.

Comment (Dark1z) [reply phpBB/docs/events.md:992]: > Doubt it's allowed, but it's up to devs.

Last time I was told to do that.

**Comment by Dark1z** [comment]
> > Twig syntax, please :)
> 
> Ahh!!! Ok...

Done !

File: phpBB/docs/events.md (hunk 1 above)
Comment (Dark1z) [reply phpBB/docs/events.md:992]: @rxu ,

Just found,
check this : https://github.com/phpbb/phpbb/pull/5571

Best regards 👍

**Review by marc1706 — APPROVED** [review]

**Pull request merged**